lib/super_digit.rb

```diff
@@ -1,15 +1,32 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity 0(n)
+# Space Complexity 0(n)
 def super_digit(n)
```

Comment:
The time complexity here is O(d^2) where d is the number of digits and the space complexity is O(1) since you're not building a list of anything.